test/cassandrax/keyspace_test.exs

```diff
@@ -0,0 +1,377 @@
+Code.require_file("../support/data_case.exs", __DIR__)
+
+defmodule TestData do
+  use Cassandrax.Schema
+
+  import Ecto.Changeset
+  import Cassandrax.Query
+
+  alias Cassandrax.TestConn
+
+  @type t :: %__MODULE__{}
+  @primary_key [:id]
+
+  table "test_data" do
+    field :id, :string
+    field :value, :string
+    field :svalue, MapSetType
+  end
+
+  def changeset(%__MODULE__{} = data, attrs \\ %{}) do
+    data
+    |> cast(attrs, [:id, :value])
+    |> validate_required([:id])
+  end
+
+  def create_table do
+    statement = [
+      "CREATE TABLE IF NOT EXISTS ",
+      "#{TestKeyspace.__keyspace__()}.test_data(",
+      "id text, ",
+      "value text, ",
+      "svalue set<text>, ",
+      "PRIMARY KEY (id))"
+    ]
+
+    {:ok, _result} = Cassandrax.cql(TestConn, statement)
+  end
+
+  def drop_table do
+    statement = "DROP TABLE IF EXISTS #{TestKeyspace.__keyspace__()}.test_data"
+
+    {:ok, _result} = Cassandrax.cql(TestConn, statement)
+  end
+end
```

Comment:
I think the reason you had to require the data case file was because you defined the TestData schema outside `Cassandrax.KeyspaceTest`.

Can you try moving it into the test module and see if you still need to require it?

Reply:
It is a bigger issue that we'll look into later.